docs/source/prgpatterns/lockstep.rst

```diff
@@ -45,16 +142,19 @@ Collective Loop
     // `frame` is a list which must be traversed collectively
     while( frame.isValid() )
     {
-        uint32_t const workerIdx = cupla::threadIdx( acc ).x;
-        using ParticleDomCfg = IdxConfig<
-            frameSize,
-            numWorker
-        >;
-        ForEachIdx< ParticleDomCfg > forEachParticle( workerIdx );
-        forEachParticle(
-           [&]( uint32_t const linearIdx, uint32_t const idx )
+        uint32_t const workerIdx = cupla::threadIdx(acc).x;
```

Comment:
Could you add a comment that here we assume 1d indexing of threads inside block?